ang/civicase/case/details/people-tab/directives/case-details-people-tab.directive.js

```diff
@@ -240,13 +240,25 @@
         message: ts('Remove %1 as %2?', { 1: role.display_name, 2: role.role })
       }).on('crmConfirm:yes', function () {
         var apiCalls = [unassignRoleCall(role)];
+
+        // when client
+        if (!role.relationship_type_id) {
+          apiCalls.push(['Relationship', 'get', {
+            case_id: item.id,
+            is_active: 1,
+            contact_id_a: role.contact_id,
+            'api.Relationship.create': { is_active: 0, end_date: 'now' }
+          }]);
```

Comment:
🍰 It is difficult to read what these API calls are doing, specially because of the chain: it starts with a `get` and ends with a `create` that is supposed to be used to update the existing relationships. Can we return this from a function that explains what is going on?

```js
apiCalls.push(getUnassignRelationshipsApiParams(role));
```

Or a better name is accepted.